.drone.star

```diff
@@ -85,9 +85,11 @@ config = {
                 "apiLocks",
                 "apiSearch1",
                 "apiSearch2",
-                "apiSharingNg",
+                "apiSharingNg1",
+                "apiSharingNg2",
                 "apiSharingNgShareInvitation",
-                "apiSharingNgLinkShare",
+                "apiSharingNgLinkSharePermission",
+                "apiSharingNgLinkShareRoot",
```

Comment:
```suggestion
                "apiSharingNgPermissions",
                "apiSharingNgRoot",
```

Reply:
But doing this, won't it make a little bit difficult to distinguish for link share test scenarios? Like: using the suites name `apiSharingNgLinkSharePermission` makes it more clear that it is related to link share. :grin:

Reply:
If the tests are only link share then it's good :+1: